### PR TITLE
Add Rails 6 support

### PR DESCRIPTION
**Pull request by mokpro**
🍏 

Adds support to rails 6.1.x

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+  # install postgres
+  - sudo apt-get install postgresql-client-$PG_VERSION postgresql-server-dev-$PG_VERSION
+  # setup pg_dump
+  - ls -al /usr/lib/postgresql/
+  - sudo ln -sfn /usr/lib/postgresql/$PG_VERSION/bin/pg_dump /usr/bin/pg_dump
+  # start up the specific version of PG
+  - sudo -E sh -c 'service postgresql stop'
+  - sleep 5s
+  - sudo -E sh -c 'service postgresql start $PG_VERSION'
+  - psql -U postgres -d postgres -c 'select setting from pg_settings where name = $m$server_version$m$;'
```
Comment (packrat386) [inline .travis.yml:23]: Do we need all this setup?

Comment (mokpro) [reply .travis.yml:23]: When I tried to build with existing config, it fails - https://travis-ci.org/github/enova/obvious_data/jobs/742080832

Let me try using [travis-ci services](https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version)

Comment (Slania) [reply .travis.yml:23]: Yeah, this was our 6 years ago workaround for installing specific PostgreSQL versions before Travis officially supported it as a regular dependency.

Comment (mokpro) [reply .travis.yml:23]: Makes sense. I'll make this a draft PR and work on it. Travis CI queue is painfully long.

I also tried to move to [github actions](https://github.com/mokpro/obvious_data/pull/2/checks?check_run_id=1764927199), but I'm blocked. Please take a look and suggest something I can try.

Comment (mokpro) [reply .travis.yml:23]: Thank you for comment @packrat386. Fixed it in https://github.com/enova/obvious_data/pull/4

Closing current PR

**Comment by mokpro** [comment]
closing this in favor of https://github.com/enova/obvious_data/pull/4

**Pull request closed without merging**